polkadot/xcm/pallet-xcm/src/tests/assets_transfer.rs

```diff
@@ -2088,14 +1978,7 @@ fn reserve_transfer_assets_with_teleportable_asset_disallowed() {
 			fee_index as u32,
 			Unlimited,
 		);
-		assert_eq!(
-			res,
-			Err(DispatchError::Module(ModuleError {
-				index: 4,
-				error: [2, 0, 0, 0],
-				message: Some("Filtered")
-			}))
-		);
+		assert_eq!(res, Err(crate::Error::<Test>::Filtered.into()));
```

Comment:
There is also `assert_err!`

Reply:
@ggwpez  Resolved in this commit https://github.com/paritytech/polkadot-sdk/pull/4248/commits/9d552711d9a2c95be9478786bf5665a4113be639. Thank you for feedback!